notebook/static-src/base/js/namespace.js

```diff
@@ -0,0 +1,70 @@
+// Copyright (c) Jupyter Development Team.
+// Distributed under the terms of the Modified BSD License.
+
+"use strict";
+
+if (!window.Jupyter) {
+    var Jupyter = {};
+    Jupyter.version = "4.1.0.dev";
```

Comment:
Hmmm... This makes me wonder if we can read this from a canonical place since we build our javascript now.